Gemfile

```diff
@@ -3,81 +3,155 @@ source 'https://rubygems.org'
 ruby '>= 2.4.4'
 
 # ------------------------------------------------
-#    RAILS
+# RAILS
+# Full-stack web application framework. (http://www.rubyonrails.org)
 gem 'rails', '~> 4.2.10'
+
+# Tools for creating, working with, and running Rails applications. (http://www.rubyonrails.org)
 gem 'railties'
 
+# GEMS ADDED TO HELP HANDLE RAILS MIGRATION FROM 3.x to 4.2
+# THESE GEMS HELP SUPPORT DEPRACATED FUNCTIONALITY AND WILL LOSE SUPPORT IN
+# FUTURE VERSIONS WE SHOULD CONSIDER BRINGING THE CODE UP TO DATE INSTEAD
+
+# Protect attributes from mass assignment in Active Record models (https://github.com/rails/protected_attributes)
+gem 'protected_attributes', '~> 1.1.3'
```

Comment:
For some reason this wants to add back in protected_attributes which was removed by another PR.
Might be due to the merge-order

Reply:
Yea, it's because development has updated. I'll rebase